src/Parser/Druid/Restoration/Modules/Talents/Stonebark.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+
+import StatisticBox from 'Interface/Others/StatisticBox';
+
+import SPELLS from 'common/SPELLS';
+import SpellIcon from 'common/SpellIcon';
+
+import { formatPercentage } from 'common/format';
+
+import Analyzer from 'Parser/Core/Analyzer';
+import calculateEffectiveHealing from 'Parser/Core/calculateEffectiveHealing';
+import HotTracker from 'Parser/Druid/Restoration/Modules/Core/HotTracking/HotTracker';
+
+const STONEBARK_HOT_INCREASE = 0.2;
+
+class Stonebark extends Analyzer {
+  static dependencies = {
+    hotTracker: HotTracker,
+  };
+
+  healing = 0;
+
+  constructor(...args) {
+    super(...args);
+    this.active = this.selectedCombatant.hasTalent(SPELLS.STONEBARK_TALENT.id);
+  }
+
+
+  on_byPlayer_heal(event) {
+    const spellId = event.ability.guid;
+
+    if(this.hotTracker.hotInfo[spellId] == null) {
+      this.healing += calculateEffectiveHealing(event, STONEBARK_HOT_INCREASE);
```

Comment:
I know the wording of the talent is a little ambiguous but shouldn't this increase be active only if the target has Ironbark active on them? Or is this condition somehow baked into the condition on the line above and I just don't understand?